src/index.js

```diff
@@ -22,3 +24,24 @@ const server = new ApolloServer({
 server.listen({ port: process.env.PORT }).then(() => {
   console.log(`🚀 Server ready at ${process.env.PORT}`);
 });
+
+// Connect Sequelize to Postgres database
+const Sequelize = require("sequelize");
+const sequelize = new Sequelize("alexmerecka", "alexmerecka", "password", {
+  host: "localhost",
+  dialect: "postgres",
+  pool: {
```

Comment:
you can remove pool and default to whatever sequelize defaults to

Reply:
Done